src/invariant/SponsorshipCountIsValid.cpp

```diff
@@ -43,6 +43,8 @@ getMult(LedgerEntry const& le)
         return 1;
     case CLAIMABLE_BALANCE:
         return le.data.claimableBalance().claimants.size();
+    case LIQUIDITY_POOL:
```

Comment:
Same thing about this not seeming quite right to me: we should never hit this branch.

Reply:
updated